mne_bids_pipeline/steps/preprocessing/_09_ptp_reject.py

```diff
@@ -204,16 +206,16 @@ def drop_ptp(
                 f"{ar.n_interpolate_} channels were bad (cross-validated n_interpolate "
                 f"limit; excluding globally bad and non-data channels, shown in white)."
             )
+            fig = reject_log.plot(orientation="horizontal", aspect="auto", show=False)
             report.add_figure(
-                fig=reject_log.plot(
-                    orientation="horizontal", aspect="auto", show=False
-                ),
+                fig=fig,
                 title=f"{kind} cleaning",
                 caption=caption,
                 section=title,
                 tags=tags,
                 replace=True,
             )
+            plt.close(fig)
```

Comment:
Need this one for green CIs and this PR is pretty trivial I think, so I'll go ahead and merge. Happy to make changes if you see issues though @hoechenberger